lib/aarch64-apple-darwin20.jl

```diff
@@ -0,0 +1,2747 @@
+    final_ll::Cint
+    final_pack::Cint
+    final_monotonic::Cint
+    final_resymbol::Cint
+    zrelax::NTuple{3, Cdouble}
+    nrelax::NTuple{3, Csize_t}
+    prefer_zomplex::Cint
+    prefer_upper::Cint
+    quick_return_if_not_posdef::Cint
+    prefer_binary::Cint
+    print::Cint
+    precise::Cint
+    try_catch::Cint
+    error_handler::Ptr{Cvoid}
+    nmethods::Cint
+    current::Cint
+    selected::Cint
+    method::NTuple{10, cholmod_method_struct}
+    postorder::Cint
+    default_nesdis::Cint
+    metis_memory::Cdouble
+    metis_dswitch::Cdouble
+    metis_nswitch::Csize_t
+    nrow::Csize_t
+    mark::Clong
+    iworksize::Csize_t
+    xworksize::Csize_t
+    Flag::Ptr{Cvoid}
+    Head::Ptr{Cvoid}
+    Xwork::Ptr{Cvoid}
+    Iwork::Ptr{Cvoid}
+    itype::Cint
+    dtype::Cint
+    no_workspace_reallocate::Cint
+    status::Cint
+    fl::Cdouble
+    lnz::Cdouble
+    anz::Cdouble
+    modfl::Cdouble
+    malloc_count::Csize_t
+    memory_usage::Csize_t
+    memory_inuse::Csize_t
+    nrealloc_col::Cdouble
+    nrealloc_factor::Cdouble
+    ndbounds_hit::Cdouble
+    rowfacfl::Cdouble
+    aatfl::Cdouble
+    called_nd::Cint
+    blas_ok::Cint
+    SPQR_grain::Cdouble
+    SPQR_small::Cdouble
+    SPQR_shrink::Cint
+    SPQR_nthreads::Cint
+    SPQR_flopcount::Cdouble
+    SPQR_analyze_time::Cdouble
+    SPQR_factorize_time::Cdouble
+    SPQR_solve_time::Cdouble
+    SPQR_flopcount_bound::Cdouble
+    SPQR_tol_used::Cdouble
+    SPQR_norm_E_fro::Cdouble
+    SPQR_istat::NTuple{10, Clong}
+    useGPU::Cint
+    maxGpuMemBytes::Csize_t
+    maxGpuMemFraction::Cdouble
+    gpuMemorySize::Csize_t
+    gpuKernelTime::Cdouble
+    gpuFlops::Clong
+    gpuNumKernelLaunches::Cint
+    cublasHandle::Ptr{Cvoid}
+    gpuStream::NTuple{8, Ptr{Cvoid}}
+    cublasEventPotrf::NTuple{3, Ptr{Cvoid}}
+    updateCKernelsComplete::Ptr{Cvoid}
+    updateCBuffersFree::NTuple{8, Ptr{Cvoid}}
+    dev_mempool::Ptr{Cvoid}
+    dev_mempool_size::Csize_t
+    host_pinned_mempool::Ptr{Cvoid}
+    host_pinned_mempool_size::Csize_t
+    devBuffSize::Csize_t
+    ibuffer::Cint
+    syrkStart::Cdouble
+    cholmod_cpu_gemm_time::Cdouble
+    cholmod_cpu_syrk_time::Cdouble
+    cholmod_cpu_trsm_time::Cdouble
+    cholmod_cpu_potrf_time::Cdouble
+    cholmod_gpu_gemm_time::Cdouble
+    cholmod_gpu_syrk_time::Cdouble
+    cholmod_gpu_trsm_time::Cdouble
+    cholmod_gpu_potrf_time::Cdouble
+    cholmod_assemble_time::Cdouble
+    cholmod_assemble_time2::Cdouble
+    cholmod_cpu_gemm_calls::Csize_t
+    cholmod_cpu_syrk_calls::Csize_t
+    cholmod_cpu_trsm_calls::Csize_t
+    cholmod_cpu_potrf_calls::Csize_t
+    cholmod_gpu_gemm_calls::Csize_t
+    cholmod_gpu_syrk_calls::Csize_t
+    cholmod_gpu_trsm_calls::Csize_t
+    cholmod_gpu_potrf_calls::Csize_t
+    cholmod_common_struct() = new()
+end
+
+const cholmod_common = cholmod_common_struct
+
+function cholmod_start(Common)
+    ccall((:cholmod_start, libcholmod), Cint, (Ptr{cholmod_common},), Common)
```

Comment:
Out of curiosity, is it possible for Clang.jl to emit `@ccall` instead of `ccall()`?

Reply:
Sure, with [`use_ccall_macro=true`](https://github.com/JuliaInterop/Clang.jl/blob/5b3b9cf2483af8cc34bfa108459486d92de7fb11/gen/generator.toml#L133). But that will make this package Julia 1.5+ only.